storybook/src/builders/webpack-builder.ts

```diff
@@ -4,6 +4,8 @@ import * as baseBuilder from '@storybook/builder-webpack5';
 import dedent from 'ts-dedent';
 
 export type BuilderOptions = {
+    projectDir?: string;
+    storybookCachePath?: string;
```

Comment:
`projectDir` is now provided by `main.js|ts` file

`storybookCachePath` is the path `var/cache` folder of StorybookBundle. I need this because my project uses [Single Kernel](https://symfony.com/doc/current/configuration/multiple_kernels.html) approach

Here is how I configured `framework.options` values

```js
options: {
    // 👇 Here configure the framework
    symfony: {
        projectDir: path.resolve(__dirname, '..'),
        storybookCachePath: path.resolve(__dirname, '../var/cache/etu/dev/storybook'),
        server: 'https://www.myapp.lh:8005',
        proxyPaths: [
            '/assets',
        ],
        additionalWatchPaths: [
            'assets',
        ]
    }
},
```

Reply:
Just found out that Storybook will inject a global var in `iframe.html` when building

```js
window['FRAMEWORK_OPTIONS'] = {"symfony":{"projectDir":"/application/front","storybookCachePath":"/path/to/var/cache/etu/dev/storybook","server":"https://www.myapp.lh","proxyPaths":["/dist"],"additionalWatchPaths":["/path/to/front/assets"],"projectPathAliases":{"/my/path/to/front":"/application/front"}}};
```

I'll try to prevent this or have an alternative